emoji/src/main/java/com/vanniktech/emoji/EmojiArrayAdapter.java

```diff
@@ -10,20 +10,27 @@
 import android.widget.ImageView;
 import com.vanniktech.emoji.emoji.Emoji;
 import com.vanniktech.emoji.listeners.OnEmojiClickedListener;
+import com.vanniktech.emoji.listeners.OnEmojiLongClickedListener;
+
```

Comment:
Android Studio is always trying to change it back, is it really necessary?
Out of curiosity, why does your code style differ so much from the Android Studio defaults? I have hard times getting Android Studio to not reformat 😁